onnx/version_converter/convert.h

```diff
@@ -110,6 +112,16 @@ class DefaultVersionConverter : public BaseVersionConverter {
         OpSetID(5), OpSetID(6)));
       registerAdapter(make_unique<Gemm_7_6>());
       registerAdapter(make_unique<Gemm_6_7>());
+      registerAdapter(make_unique<RemoveConsumedInputs>("Relu",
+        OpSetID(5), OpSetID(6)));
+      registerAdapter(make_unique<CompatibleAdapter>("Relu",
+        OpSetID(6), OpSetID(5)));
+      registerAdapter(make_unique<CompatibleAdapter>("BatchNormalization",
```

Comment:
This is not compatible, because default value of is_test is 0... you have to set it as 1.